modules/@apostrophecms/rich-text-widget/ui/apos/components/AposRichTextWidgetEditor.vue

```diff
@@ -753,6 +753,11 @@ function traverseNextNode(node) {
   $z-index-button-background: 1;
   $z-index-button-foreground: 2;
 
+  .bubble-menu {
+    width: max-content;
+    max-width: 95vw;
+  }
+
```

Comment:
Check that toolbar can go beyond parent width (editor) but still shrink to stay visible when window resize.